EMS/common-bundle/src/Common/CoreApi/Endpoint/Form/Form.php

```diff
@@ -32,11 +33,18 @@ public function getSubmission(string $submissionId, ?string $property = null): a
         return $this->client->get($resource, $query)->getData();
     }
 
-    public function getSubmissionFile(string $submissionId, ?string $submissionFileId): StreamedResponse
+    public function getSubmissionFile(string $submissionId, ?string $submissionFileId): ResponseInterface
```

Comment:
The getSubmissionFile method is also used in \EMS\ClientHelperBundle\Controller\ApiController::getSubmissionFile

Which now is not working with a StreamedResonse anymore?

Reply:
Yes as \EMS\ClientHelperBundle\Controller\ApiController::getSubmissionFile is now using the getSubmissionFileAsStreamResponse method